core/datastorage/src/main/java/com/cognifide/aet/vs/metadata/MetadataDAOMongoDBImpl.java

```diff
@@ -143,6 +157,21 @@ public Suite apply(Document result) {
     }).toList();
   }
 
+  public List<String> listSuiteVersions(DBKey dbKey, String name) throws StorageException{
```

Comment:
This method should return list of pairs: `{ version: "..", correlationId: "..}` - we can't assume that oldest suite will have `version` equal to `1` because Cleaner might have deleted it